src/components/StatementOfCashFlow/StatementOfCashFlow.tsx

```diff
@@ -116,6 +117,12 @@ const StatementOfCashFlowView = ({
           <Header>
             <HeaderRow>
               <HeaderCol>{datePicker}</HeaderCol>
+              <HeaderCol>
+                <CashflowStatementDownloadButton
+                  startDate={startDate}
+                  endDate={endDate}
+                />
```

Comment:
NIT: you can add the following to have icon-only download button (like on BalanceSheet)

```suggestion
import { useElementViewSize } from '../../hooks/useElementViewSize'
import { View as ViewType } from '../../types/general'

  const [view, setView] = useState<ViewType>('desktop')
  const containerRef = useElementViewSize<HTMLDivElement>(newView =>
    setView(newView),
  )

<View
  (...)
  ref={containerRef}
  >

                <CashflowStatementDownloadButton
                  startDate={startDate}
                  endDate={endDate}
                  iconOnly={view === 'mobile'}
                />
```

![image](https://github.com/user-attachments/assets/4a7a4f41-61f7-4a5f-9f38-77fd0d0873d3)

Reply:
I appreciate the catch.

I think we should be able to do even-better in the long-term with CSS-only approaches (as we still have a bit of flicker / layout shift).